Transport/BacnetIpUdpProtocolTransport.cs

```diff
@@ -30,16 +31,17 @@ public class BacnetIpUdpProtocolTransport : BacnetTransportBase
         // Some more complex solutions could avoid this, that's why this property is virtual
         public virtual IPEndPoint LocalEndPoint => (IPEndPoint)_exclusiveConn.Client.LocalEndPoint;
 
-        public BacnetIpUdpProtocolTransport(int port, bool useExclusivePort = false, bool dontFragment = false,
+        public BacnetIpUdpProtocolTransport(int sharedPort, int exclusivePort = 0, bool dontFragment = false,
             int maxPayload = 1472, string localEndpointIp = "")
         {
-            SharedPort = port;
+            SharedPort = sharedPort;
+            this.exclusivePort = exclusivePort;
+
```

Comment:
I know this part of code sucks. Having a property SharedPort but being used as exclusive port 😩 But anyway, before we rewrite this, I suggest as minimal changes as necessery. How about you add extra ctor:
```csharp
public int ExclusivePort { get; }

public BacnetIpUdpProtocolTransport(int sharedPort, int exclusivePort, bool dontFragment = false,
    int maxPayload = 1472, string localEndpointIp = "")
    : this(sharedPort, true, dontFragment, maxPayload, localEndpointIp)
{
    ExclusivePort = exclusivePort;
}
```